test/functional/interface_rest.py

```diff
@@ -114,7 +114,7 @@ def run_test(self):
         assert_equal(self.nodes[1].getbalance(), Decimal("0.1"))
 
         # Check chainTip response
-        json_obj = self.test_rest_request("/getutxos/{}-{}".format(*spending))
+        json_obj = self.test_rest_request(f"/getutxos/{spending[0]}-{spending[1]}")
```

Comment:
While f-strings are preferred, I think here it is subjective which version is better. Since our dev notes don't require a specific style, I think it is fine to not use f-strings everywhere.